servo-core/src/main/java/com/netflix/servo/monitor/Monitors.java

```diff
@@ -15,8 +15,7 @@
  */
 package com.netflix.servo.monitor;
 
-import com.google.common.base.Function;
-import com.google.common.cache.Cache;
+import com.github.benmanes.caffeine.cache.Cache;
```

Comment:
If there is no usage of `Monitors.newCacheMonitor` internally at Netflix that would break, then I would prefer to just remove the method rather than move it over to Caffeine.

Reply:
Makes sense to me, will remove vs update to Caffeine 👍

Reply:
Looks like DynamicGauge uses it 🤔